eventsource-client/src/client.rs

```diff
@@ -404,7 +404,7 @@ where
     type Item = Result<SSE>;
 
     fn poll_next(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
-        trace!("ReconnectingRequest::poll({:?})", &self.state);
+        trace!(target: "ldeventsource", "ReconnectingRequest::poll({:?})", &self.state);
```

Comment:
I think the big question is:

1. Do we consider this a breaking change since others could have potentially configured their logs to filter? 
2. What target do we want here? We are making the same change in the SDK so as consistent as possible would be nice. We could just use the crate name directly (though it is eventsource-client, and not ld prefixed).

Thoughts?

Reply:
1. I think it is breaking, but the crate isn't 1.0, so I don't think it is a problem.

I have lots of questions about target, because it sounds like this would be more dynamic by default.